src/main/java/com/softserve/controller/home/HomeController.java

```diff
@@ -1,11 +1,19 @@
 package com.softserve.controller.home;
 
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import com.softserve.service.SubCategoryService;
 
 @Controller
 public class HomeController {
 
+	private SubCategoryService articleService;
+
+	public HomeController(SubCategoryService articleService){
```

Comment:
instead of doing this (initializing through the constructor), you can use @Autowired

```
@Autowired
private SubCategoryService articleService;
```